ios/ImageCropPicker.m

```diff
@@ -363,29 +381,45 @@ - (void) getVideoAsset:(PHAsset*)forAsset completion:(void (^)(NSDictionary* ima
          NSString *filePath = [tmpDirFullPath stringByAppendingString:[[NSUUID UUID] UUIDString]];
          filePath = [filePath stringByAppendingString:@".mp4"];
          NSURL *outputURL = [NSURL fileURLWithPath:filePath];
-
-         [self.compression compressVideo:sourceURL outputURL:outputURL withOptions:self.options handler:^(AVAssetExportSession *exportSession) {
-             if (exportSession.status == AVAssetExportSessionStatusCompleted) {
-                 AVAsset *compressedAsset = [AVAsset assetWithURL:outputURL];
-                 AVAssetTrack *track = [[compressedAsset tracksWithMediaType:AVMediaTypeVideo] firstObject];
-
-                 NSNumber *fileSizeValue = nil;
-                 [outputURL getResourceValue:&fileSizeValue
-                                      forKey:NSURLFileSizeKey
-                                       error:nil];
-
-                 completion([self createAttachmentResponse:[outputURL absoluteString]
-                                                 withLocalIdentifier: forAsset.localIdentifier
-                                                 withFilename: sourceURL.lastPathComponent
-                                                 withWidth:[NSNumber numberWithFloat:track.naturalSize.width]
-                                                withHeight:[NSNumber numberWithFloat:track.naturalSize.height]
-                                                  withMime:@"video/mp4"
-                                                  withSize:fileSizeValue
-                                                  withData:[NSNull null]]);
-             } else {
-                 completion(nil);
-             }
-         }];
+         
+         if ([self.options[@"compressVideoPreset"]  isEqual: @"Original"]) {
+             //if the compress video preset is set to be original, then do not compress the video
+             NSLog(@"video info: %@", info);
+             AVAssetTrack *track = [[asset tracksWithMediaType:AVMediaTypeVideo] firstObject];
+             completion([self createAttachmentResponse:@""
+                                   withLocalIdentifier:forAsset.localIdentifier
+                                          withFilename:sourceURL.lastPathComponent
+                                             withWidth:[NSNumber numberWithFloat:track.naturalSize.width]
+                                            withHeight:[NSNumber numberWithFloat:track.naturalSize.height]
+                                              withMime:@"video/quicktime"
```

Comment:
I think we cannot hardcode mime type here, because not always returned video is quicktime. Video can also be mp4 if it is sent via whats app for example. Is there some other way to find a mime type?